url/urlsearchparams-sort.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<meta charset="utf8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+[
+  {
+    "input": "z=b&a=b&z=a&a=a",
+    "output": [["a", "b"], ["a", "a"], ["z", "b"], ["z", "a"]]
+  },
+  {
+    "input": "\uFFFD=x&\uFFFC&\uFFFD=a",
+    "output": [["\uFFFC", ""], ["\uFFFD", "x"], ["\uFFFD", "a"]]
+  }
+].forEach((val) => {
```

Comment:
I think we need at least a few tests on Unicode inputs, to show what type of sorting is being applied here. I assume it's just going to be by character code, so instead of sorting é near e, it'll sort after the usual alphabet? Tests to confirm.